BrightCoders-Institute/s5a1-administrador-de-eventos-team4
# Create map to save & show ubication

## Quick Info
   - Map to save the ubication where you click
   - Search button tool
   - Show map with the ubication saved when you edit an event
   - Navigate inside the map with the cursor
   - FIll automatically ubication fields on the form  (ubicacion, latitud, longitud)

Migrations? 
Yes, added latitud & longitud columns

## What does this change?
The way to save & show the ubication

## Why are you changing that?
Because it looks better with a interactive map and makes a better experience for the user 

## How do you manually test this?
Clicking many times on the map to see if the infomation was showed on the ubication fields.
Saving events and see the correct ubication info on my view
Editing events to see if the ubicatiion saved was showed on the map

## Screenshots
![add_event](https://github.com/BrightCoders-Institute/s5a1-administrador-de-eventos-team4/assets/86389712/918900fe-797c-4245-92a3-2241c6a60239)
![show_and_edit_ubi](https://github.com/BrightCoders-Institute/s5a1-administrador-de-eventos-team4/assets/86389712/dbf1fa08-ba9b-43f0-8cff-0ae7f268d96d)
